src/main/java/net/ccbluex/liquidbounce/features/module/modules/world/Scaffold.kt

```diff
@@ -760,10 +778,13 @@ object Scaffold : Module("Scaffold", ModuleCategory.WORLD, Keyboard.KEY_I) {
 
         var rotation = toRotation(vec, false)
 
-        rotation = if (stabilizedRotation) {
+
+        rotation = if (rotationMode == "Stabilized") {
             Rotation(round(rotation.yaw / 45f) * 45f, rotation.pitch)
-        } else {
+        } else if (rotationMode == "Normal") {
             rotation
+        } else {
+            Rotation(mc.thePlayer.rotationYaw + 180f, rotation.pitch)
```

Comment:
Backwards mode makes no sense, placing blocks while facing opposite direction of players client-side rotation cannot bypass anything, it should rotate opposite to the EnumFacing instead.

Reply:
How do I do that?

Reply:
> Backwards mode makes no sense, placing blocks while facing opposite direction of players client-side rotation cannot bypass anything, it should rotate opposite to the EnumFacing instead.

Actually, it does bypass multiple anti-cheats (such as Intave), and it's pretty good.